src/core/utils/index.js

```diff
@@ -299,3 +300,8 @@ export function addonHasVersionHistory(addon) {
     ADDON_TYPE_THEME,
   ].includes(addon.type);
 }
+
+export const stripHtmlEntities = (string) => {
+  const entities = new AllHtmlEntities();
```

Comment:
Interesting... previously [we just used `sanitizeHTML()` with no allowed tags](https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/Collection/index.js) to filter HTML entities. That's a bit more of an obscured use of that helper but it means we didn't need an extra library.

I'd be tempted to just make this helper that use `text => sanitizeHTML(text, [])` under the covers to allow HTML entities. That way we don't have to include and bundle an extra library. If the tests passed when this helper looked like:

```js
export const stripHtmlEntities = (string) => {
  return sanitizeHTML(string);
}
```

then I think that would be the better route to go.